src/bio/bio_buffer.c

```diff
@@ -357,9 +359,13 @@ chunk_get_idle(struct bio_dma_buffer *bdb, struct bio_desc *biod)
 	if (d_list_empty(&bdb->bdb_idle_list)) {
 		if (bdb->bdb_tot_cnt == bio_chk_cnt_max) {
 			D_CRIT("Maximum per-xstream DMA buffer isn't big "
-			       "enough (chk_sz:%u chk_cnt:%u iods:%u) to "
-			       "sustain the workload.\n", bio_chk_sz,
-			       bio_chk_cnt_max, bdb->bdb_active_iods);
+			       "enough (chk_sz:%u chk_cnt:%u iods:%u "
+			       "used:%u,%u,%u) to sustain the workload.\n",
+			       bio_chk_sz, bio_chk_cnt_max,
+			       bdb->bdb_active_iods,
+			       bdb->bdb_used_cnt[BIO_CHK_TYPE_IO],
+			       bdb->bdb_used_cnt[BIO_CHK_TYPE_LOCAL],
+			       bdb->bdb_used_cnt[BIO_CHK_TYPE_REBUILD]);
```

Comment:
probably change to DB_REBUILD for REBUILD_TYPE,  since performance is less sensitive for rebuild anyway.

Reply:
If we used "DB_REBUILD" or other DEBUG mask at the very beginning, the two issues (1. rebuild crazily creates pull ULTs. 2. Mixed requests within same chunk generates severe fragmentations) could be overlooked.

I agree that if the DMA shortage situation is _really_ caused by rare peak workload generated by client IO, a less severe debug level should be more proper.

I'll make it into a D_INFO and put it into telemetry in later patches.